### PR TITLE
Dynamic link for Rate the App

### LINKED ISSUE #554: Custom DWDS apps “Rate the App” link is wrong
image

Could we make this link dynamic? Based on app ID?

### PR DESCRIPTION
**Pull request by BPerlakiH**
Fixes #554
Blocked by #520

### CODE REVIEW

File: Views/Settings/Settings.swift [hunk 1]
```diff
@@ -216,7 +216,8 @@ struct Settings: View {
         Section("Misc".localized) {
             Button("Feedback".localized) { UIApplication.shared.open(URL(string: "mailto:feedback@kiwix.org")!) }
             Button("Rate the App".localized) {
-                let url = URL(string: "itms-apps://itunes.apple.com/us/app/kiwix/id997079563?action=write-review")!
```
Comment (tvision251) [inline Views/Settings/Settings.swift:219]: @BPerlakiH Please replace the App store ID using `Brand.appstoreId`

**Comment by kelson42** [comment]
@BPerlakiH can you please rebase on git main head?

File: Views/Settings/Settings.swift (hunk 1 above)
Comment (BPerlakiH) [reply Views/Settings/Settings.swift:219]: @tvision251 I am not sure if I understand your point here. 
With these changes the value from Brand.appStoreId is used.

**Review by tvision251 — APPROVED** [review]

**Comment by BPerlakiH** [comment]
@kelson42 Please see the direct changes to the Kiwix/Apple repo here (including these ones): 
https://github.com/kiwix/apple/pull/569

**Comment by BPerlakiH** [comment]
this PR not to be merged directly

**Comment by BPerlakiH** [comment]
No longer needed

**Pull request closed without merging**